OWNERS

```diff
@@ -1,16 +1,9 @@
-maintainers:
-  - adamreese
-  - bacongobbler
-  - jascott1
-  - michelleN
-  - migmartri
-  - nebril
-  - prydonius
-  - seh
-  - technosophos
-  - thomastaylor312
-  - vaikas-google
-  - viglesiasce
+# See the OWNERS docs: https://git.k8s.io/community/docs/devel/owners.md
+
+approvers:
+  - sig-apps-leads
```

Comment:
Can we remove this alias and the alias from OWNERS_ALIASES? leaders from sig-apps do not have the right to approve helm PRs at this time. We just have two groups: core maintainers and reviewers.

I will note that this may change in the future with https://github.com/kubernetes/helm/issues/2792, but at this time SIG Apps leads do not have direct approval rights. We just report upstream to the weekly SIG Apps meetings.

On another note, it might be better to reference SIG Apps' OWNERS file here so that we get up-to-date changes when SIG Apps Leadership changes over time. That way there's less code duplication overall.

Thanks!

Reply:
>is there a way we can reference a SIG Apps' OWNERS file here instead 

DRY teams isn't a today thing.  But on my radar.

Reply:
SIG Apps leads should not be in the owners files. Per the [community membership process](https://github.com/kubernetes/community/blob/master/community-membership.md), one needs to earn the right to have approver level and the leads have not done that by their role as lead.

Can we just remove the idea of `sig-apps-leads` from this whole PR?